sci-electronics/kicad/kicad-5.0.0.ebuild

```diff
@@ -0,0 +1,130 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+
+WX_GTK_VER="3.0"
+
+inherit check-reqs cmake-utils eapi7-ver eutils flag-o-matic gnome2-utils python-single-r1 wxwidgets vcs-snapshot xdg
+
+SERIES=$(ver_cut 1-2)
```

Comment:
do not introduce useless variables for a single use, embed ver_cut directly

Reply:
It's more for readability and easier maintenance. But if this is the new guideline, I can move it to direct embedded use.